LICENSE-MIT

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2019 SeKey
```

Comment:
If you have completely rewritten the code to where none of the original exists, you can remove their original copyright as well